PipeableTests/PipeableElement_ClickTests.swift

```diff
@@ -24,6 +24,80 @@ final class PipeableElementClickTests: PipeableXCTestCase {
         }
     }
 
+    func testBubbles() async throws {
```

Comment:
A bit of a nit, but could we pick a more descriptive name for the test -- what are we testing re: bubbling -- that the correct events bubble and the others dont? That's fine, but re: other comment is this what is being tested here?

Reply:
Totally my bad, renamed the test and added a detailed comment (see above).